src/main/camhook/cam.c

```diff
@@ -706,6 +706,11 @@ void fill_cam_struct(struct CameraData *data, const char *devid)
         // Device instance path
         strcpy(data->deviceInstancePath, devid);
         // continue
+    } else if (check_four(devid, "SKIP")) {
```

Comment:
I am a bit on the fence for using the string `SKIP` here as the implied type for this field is a device path. Mixing it's purpose with a new type of feature switch makes this switch more complex and blurry to understand.

Instead, I suggest to add an explicit new option in [config-cam](https://github.com/djhackersdev/bemanitools/blob/e9d24f7f90d9e60e6a922ea969fb56b89a7d9914/src/main/camhook/config-cam.h#L13) that allows you to "disable" specific cameras (as far as I understand the idea of "unassigned" here and what it does for the end-user).

```c
struct camhook_config_cam {
    bool disable_emu;
    size_t num_devices;
    char device_id[CAMHOOK_CONFIG_CAM_MAX][MAX_PATH];
    bool disable_cam;
};
```

Reply:
It would need to be an array, but that seems reasonable to me, would probably simplify the check later too.